package.json

```
@@ -23,7 +24,7 @@
     "reinstall": "rm -rf node_modules/ && yarn cache clean && yarn"
   },
   "dependencies": {
-    "@pooltogether/draw-calculator-js": "^1.0.0-beta.24",
+    "@pooltogether/draw-calculator-js": "^1.0.0-beta.25",
```

Would be preferable to install fixed version without the `ˆ`. Same remark for all the other dependencies.
```suggestion
    "@pooltogether/draw-calculator-js": "1.0.0-beta.25",
```